src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public Truck() {
```

Comment:
This constructor is redundant, because default empty constructor will created anyway, you don't need to declare it explicitely